tests/test_habitat_quality.py

```diff
@@ -2105,3 +2114,5 @@ def test_habitat_quality_validate_missing_fut_column(self):
                 header='column', header_name='fut_path')
         )]
         self.assertEqual(validate_result, expected)
+
+# @TODO: ¿add rarity CSV test?
```

Comment:
Thinking of adding an additional test!?

Reply:
Oops! I was, but then I realized it'd be simpler to update an existing test instead.